gallery/src/components/NxToast/NxToastFullPageExample.tsx

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { useState } from 'react';
+import {reject, propEq} from 'ramda';
+import {
+  NxGlobalSidebarNavigation,
+  NxGlobalSidebarNavigationLink,
+  NxPageSidebar,
+  NxButton,
+  NxFontAwesomeIcon,
+  NxBackButton,
+  NxStatefulGlobalSidebar,
+  NxPageMain,
+  NxPageTitle,
+  NxH1,
+  NxToastContainer,
+  NxToast,
+  NxSuccessAlert,
+  NxTile,
+  NxP
+} from '@sonatype/react-shared-components';
+
+import {ToastModel} from '@sonatype/react-shared-components/components/NxToast/types';
+
+import {
+  faArrowLeft,
+  faArrowRight,
+  faLink,
+  faQuestionCircle,
+  faBell,
+  faCog,
+  faUserCircle
+} from '@fortawesome/free-solid-svg-icons';
+
+const sidebarLogoPath = require('../../assets/images/logo-plaid-villain-text.png');
+
+export default function NxToastFullPageExample() {
+
+  const [toasts, setToasts] = useState<ToastModel[]>([]);
+  const [id, setId] = useState<number>(0);
+
+  const addToast = () => {
+    setId(id + 1);
+    setToasts((toasts) => [
+      { toastId: id },
+      ...toasts
+    ]);
+  };
+
+  const removeToast = (id:number) => {
+    const _toasts = reject(propEq('toastId', id), toasts);
+    setToasts(_toasts);
+  };
+
+  return (
+    <>
+      <header className="nx-global-header">
+        <NxBackButton href="#/pages/Toast" targetPageTitle="Documentation" />
+        <div className="nx-global-header__actions">
+          <NxButton title="Help" variant="icon-only"><NxFontAwesomeIcon icon={faQuestionCircle} /></NxButton>
+          <NxButton title="Notifications" variant="icon-only"><NxFontAwesomeIcon icon={faBell} /></NxButton>
+          <NxButton title="Settings" variant="icon-only"><NxFontAwesomeIcon icon={faCog} /></NxButton>
+          <NxButton title="User" variant="icon-only"><NxFontAwesomeIcon icon={faUserCircle} /></NxButton>
+        </div>
+      </header>
+      <NxStatefulGlobalSidebar isDefaultOpen={false}
+                               toggleOpenIcon={faArrowLeft}
+                               toggleCloseIcon={faArrowRight}
+                               logoImg={sidebarLogoPath}
+                               logoAltText="RSC Plaid Villain"
+                               logoLink="#">
+        <NxGlobalSidebarNavigation>
+          <NxGlobalSidebarNavigationLink icon={faLink} text="NxGlobalSidebar" href="#/pages/NxGlobalSidebar"/>
+        </NxGlobalSidebarNavigation>
+      </NxStatefulGlobalSidebar>
+      <NxPageSidebar tabIndex={0}>
```

Comment:
This example has a bunch of parts that aren't really important to the point.  I suggest simplifying it.  The header should probably be present but can be empty (though you'll need another example without the header too, to prove how toasts lay out in that scenario).  The `NxPageSidebar` can be removed.  IMO `NxPageMain` should be given enough content that it's likely to scroll, so that the interaction of that scrolling with the toasts can be tested/demonstrated.

Reply:
> This example has a bunch of parts that aren't really important to the point. I suggest simplifying it. The header should probably be present but can be empty (though you'll need another example without the header too, to prove how toasts lay out in that scenario). The `NxPageSidebar` can be removed. IMO `NxPageMain` should be given enough content that it's likely to scroll, so that the interaction of that scrolling with the toasts can be tested/demonstrated.

I removed NxPageSidebar, added in content and the button bar on the bottom for scrolling. (some of the code is outdated here)-[c1e07de](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/c1e07de714219ad666458562a5b9e494da737abd)